.github/workflows/bit32-multi-arch-tests.yml

```diff
@@ -38,6 +38,7 @@ jobs:
               echo -e "\033[0m" >&2
               "$@"
             }
+            export CC=clang
```

Comment:
Why with clang?

Reply:
I was getting compiler crashes (as in `gcc` "internal compiler error" bugs) on non-x86 archs: https://github.com/lunarmodules/lua-compat-5.3/actions/runs/13035598909/job/36365131177